src/components/team/copy-to-clipboard.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react'
```

Comment:
The Team page file was getting large, so I extracted this to a standalone component to reduce the noise and remove clear up some imports.